applications/luci-app-upnp/luasrc/view/upnp_status.htm

```diff
@@ -23,6 +23,7 @@
 						st[i].proto,
 						st[i].extport,
 						st[i].intaddr,
+						st[i].intaddr_resolved,
```

Comment:
Maybe consider adding an ` || '-'` here since there might not always be a value available